scalding-core/src/main/scala/com/twitter/scalding/TemplateSource.scala

```diff
@@ -36,12 +35,12 @@ abstract class TemplateSource extends SchemedSource with HfsTapProvider {
   def pathFields: Fields = Fields.ALL
 
   /**
-   * Creates the template tap.
+   * Creates the partition tap.
    *
    * @param readOrWrite Describes if this source is being read from or written to.
    * @param mode The mode of the job. (implicit)
    *
-   * @returns A cascading TemplateTap.
+   * @return A cascading PartitionTap.
```

Comment:
are you sure this works? Unfortunately, I think this may be untested. If you want to change this code, we should point to the test.

I'm nervous about changing untested code without adding tests.